src/ILCompiler/repro/Program.cs

```diff
@@ -4,10 +4,26 @@
 
 using System;
 
-internal class Program
+internal class Program:IInterface
```

Comment:
The changes in this file should be rolled back.

To verify that your changes work well, you may remove `#if CORERT` todo in Encoding.cs; or tests\src\Simple\Interfaces\no_cpp tag on the interfaces unit test.